scraper_hospital_data.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
```

Comment:
This file should also be marked as executable (like the `.sh` file is), otherwise this line is kinda pointless. 😉

Reply:
Aha, of course! Thanks. I've made it executable.